hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -75,6 +75,33 @@ if {$PARAMS != ""} {
   elaborate $DUT
 }
 
+#################################
+## Define Common Synchronizers ##
+#################################
+
+# Glitch Free Mux
+# WARNING!!! prim_clock_mux2 is not a glitch free mux
+#set_user_glitch_free_muxes -name opentitan_clock_mux prim_generic_clock_mux2
+#set_user_glitch_free_muxes -name opentitan_glitchfree_mux prim_generic_clock_glitchfree_mux2
+
+# 2FF synchronizer.
+# TODO: Process dependent module name later
+set prim_2ff_modules {}
+
+# Find every derivated modules from 2FF synchronizer
+foreach mod [get_all_modules prim_flop_2sync] {
+  lappend prim_2ff_modules $mod
+  puts "Adding to list prim_2ff_modules: $mod"
+}
+
+set_user_cntl_synchronizer -name opentitan_2ff $prim_2ff_modules
```

Comment:
o this sucks, you have to use it like this?? 
I was hoping we could get away just using the module name... i guess this is more like timing where things work off of instance names instead.

Reply:
Looks like the tool can recognize 2FF synchronizer at least. But not the FIFO ..

Reply:
do we want to make any suggestions to the tool vendor? :)
i feel like if they can't recognize the fifo...then at least they should at least allow us to specify it... or maybe they're expecting us to do a bottom up approach..